sentry-core/src/clientoptions.rs

```diff
@@ -40,6 +40,24 @@ pub enum SessionMode {
     Request,
 }
 
+/// The maximum size of an HTTP request body that the SDK captures.
+///
+/// Only request bodies that parse as JSON or form data are currently captured.
+/// See the [Documentation on attaching request body](https://develop.sentry.dev/sdk/expected-features/#attaching-request-body-in-server-sdks)
+/// and the [Documentation on handling sensitive data](https://develop.sentry.dev/sdk/expected-features/data-handling/#sensitive-data)
+/// for more information.
+#[derive(Clone, PartialEq)]
```

Comment:
This could probably be `Copy` and `Default`.

Also, another variant with `Explicit(usize)` would also be nice.

It would also be nice if this enum had a `len()` method or similar defined right on the enum.
That way, the actual size limit is close to the enum and its documentation, and not in a completely different part of the code.

Reply:
Thanks for the feedback! I've added the `Explicit` variant and created an `is_within_size_limit` method on the enum to keep the size validation logic close to its definition.

Regarding `Copy` - I kept `Clone` because it's required by `ClientOptions`. If you know a way to make it work with `Copy`, I'm open to suggestions.

Reply:
You can make this `Copy` independent of the `ClientOptions`.